build/__javaVersions.sh

```diff
@@ -1,4 +1,5 @@
 # This file was auto-generated from 'constants.yaml'. Changes may be overridden.
 
 JAVA_VERSION='17.0.2'
+JAVA_VERSION11='11.0.14'
```

Comment:
Could you please clarify why we need to add this field? Why do we need it for these language and not for other languages where we support multiple major versions?

Reply:
Yes, this is because this got removed from the dev storage account. Which blocked our pipelines. As a result, we had to re-build this and update the source for rebuilding.

Reply:
OK thanks. So I imagine we'll remove it in the future once we rebuilt it?

Reply:
I believe we need to keep it as I'm using it here https://github.com/microsoft/Oryx/blob/8c8aa3034a9e46523af1154a901717983a7400e6/images/build/Dockerfiles/vso.focal.Dockerfile#L242

Reply:
Yes, we'll keeping this since its an active version